schema/crdb/7.0.0/up5.sql

```diff
@@ -0,0 +1,6 @@
+-- copy existing ip_pool-to-silo assocations into association table
+INSERT INTO ip_pool_resource (ip_pool_id, resource_type, resource_id, is_default)
+SELECT id, 'silo', silo_id, is_default
+FROM ip_pool
+WHERE silo_id IS NOT null
+  AND time_deleted IS null;
```

Comment:
@smklein `integration_tests::schema::versions_have_idempotent_up` is mad at me for `up4` and `up5` because the `is_default` and `silo_id` columns get deleted from `ip_pool` in `up7`, so these inserts fail when run a second time. I can't figure out how to make this insert conditional on those columns existing. Moving `up6` and `up7` into their own separate migration 8.0.0 migration makes the test pass. Is that good, though? Almost feels like too easy of a workaround.